projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/DeploymentServiceV2.java

```diff
@@ -0,0 +1,176 @@
+    //  return;
+    // }
+
+    JobDeployment jobDeployment = new JobDeployment();
+    jobDeployment.setDataJobTeam(dataJob.getJobConfig().getTeam());
+    jobDeployment.setDataJobName(dataJob.getName());
+
+    jobDeployment.setEnabled(dataJobDeployment.getEnabled());
+
+    DataJobResources dataJobResources = new DataJobResources();
+    dataJobResources.setCpuLimit(dataJobDeployment.getResourcesCpuLimit());
+    dataJobResources.setCpuRequest(dataJobDeployment.getResourcesCpuRequest());
+    dataJobResources.setMemoryLimit(dataJobDeployment.getResourcesMemoryLimit());
+    dataJobResources.setMemoryRequest(dataJobDeployment.getResourcesMemoryRequest());
+
+    jobDeployment.setResources(dataJobResources);
+    jobDeployment.setGitCommitSha(dataJobDeployment.getGitCommitSha());
+    jobDeployment.setPythonVersion(dataJobDeployment.getPythonVersion());
+
+    try {
+      log.info("Starting deployment of job {}", jobDeployment.getDataJobName());
+      deploymentProgress.started(dataJob.getJobConfig(), jobDeployment);
+
+      if (jobDeployment.getPythonVersion() == null) {
+        jobDeployment.setPythonVersion(supportedPythonVersions.getDefaultPythonVersion());
+      }
+
+      String imageName =
+          dockerRegistryService.dataJobImage(
+              jobDeployment.getDataJobName(), jobDeployment.getGitCommitSha());
+      jobDeployment.setImageName(imageName);
+
+      if (jobImageBuilder.buildImage(imageName, dataJob, jobDeployment, sendNotification)) {
+        log.info(
+            "Image {} has been built. Will now schedule job {} for execution",
+            imageName,
+            dataJob.getName());
+        jobDeployment.setImageName(imageName);
+        if (jobImageDeployer.scheduleJob(
+            dataJob,
+            jobDeployment,
+            sendNotification,
+            dataJobDeployment.getLastDeployedBy(),
+            cronJobs)) {
+          log.info(
+              String.format(
+                  "Successfully updated job: %s with version: %s",
+                  jobDeployment.getDataJobName(), jobDeployment.getGitCommitSha()));
+
+          saveDeployment(dataJob, jobDeployment);
+
+          deploymentProgress.completed(dataJob, sendNotification);
+        }
+      }
+    } catch (ApiException e) {
+      handleException(dataJob, jobDeployment, sendNotification, new KubernetesException("", e));
+    } catch (Exception e) {
+      handleException(dataJob, jobDeployment, sendNotification, e);
+    } catch (Throwable e) {
+      handleException(dataJob, jobDeployment, sendNotification, e);
+      throw e;
+    }
+  }
+
+  private void saveDeployment(DataJob dataJob, JobDeployment jobDeployment) {
+    // Currently, store only 'enabled' in the database
+    if (!Objects.equals(dataJob.getEnabled(), jobDeployment.getEnabled())) {
+      dataJob.setEnabled(jobDeployment.getEnabled());
+      jobsRepository.save(dataJob);
+
+      log.info(
+          "The deployment of the data job {} has been {}",
```

Comment:
This log statement does not make sense? What is going to print out?

Reply:
`The deployment of the data job data_job_name has been ENABLED/DISABLED`?